src/DssSpell.t.sol

```diff
@@ -540,73 +515,357 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(arbitrumGateway.validDomains(arbDstDomain), 0, "l2-arbitrum-invalid-dst-domain");
     }
 
-    function testDirectSparkIntegration() public {
+    // RWA tests
+
+    function sub(uint x, uint y) internal pure returns (uint z) {
```

Comment:
This is solc 0.8, no need for custom safe math.

Reply:
Seconding this, for `daiPsmGemDiffDecimals` we can change it from 

`10**sub(dai.decimals() - psmGem.decimals());`

To something more like

`10 ** (dai.decimals() - psmGem.decimals());`

Just check the ordering is correct for x and y (like what corresponds to what)